TESTING.md

```diff
@@ -24,15 +24,28 @@ before writing new integration tests, which are slower to run and are not as eas
 Before running tests, make sure your Flutter SDK matches the version that will be used on
 the CI. To update your local flutter version, run:
 
+```shell
+flutter channel main
+flutter upgrade
 ```
-./tool/update_flutter_sdk.sh --local
-```
+
 > Note: Running this command requires that you have followed the [set up instructions](CONTRIBUTING.md#set-up-your-devtools-environment)
 in the DevTools contributing guide regarding cloning the Flutter SDK from GitHub.
 
-Now you can proceed with running DevTools tests:
+Do some one-time configuration of the packages and generate testing mocks:
```

Comment:
replace with:

You may need to re-generate the testing mocks before running the tests:
```
devtools_tool generate-code
```


(we don't need the flutter pub get commands because this should happen automatically with the generate-code command)

Reply:
When I run this without doing the pub get commands, I get an error where it tries to access `.dart_tool/package_config.json` in the `devtools_app` folder:

```
(code/devtools:upgrade_riverpod) -- devtools > devtools_tool generate-code
Using Flutter SDK from /usr/local/google/home/gspencer/code/devtools/tool/flutter-sdk
Running build_runner build for devtools_app
/usr/local/google/home/gspencer/code/devtools/packages/devtools_app > /usr/local/google/home/gspencer/code/devtools/tool/flutter-sdk/bin/flutter pub run build_runner build --delete-conflicting-outputs
Downloading Material fonts...                                      316ms
Downloading Gradle Wrapper...                                       69ms
Downloading package sky_engine...                                  259ms
Downloading package flutter_gpu...                                  87ms
Downloading flutter_patched_sdk tools...                           542ms
Downloading flutter_patched_sdk_product tools...                   475ms
Downloading linux-x64 tools...                                   1,034ms
Downloading linux-x64/font-subset tools...                         188ms
Cannot open file, path = '.dart_tool/package_config.json' (OS Error: No such file or directory, errno = 2)
Failed to update packages.
ProcessException: Failed with exit code: 66. 
  Command: /usr/local/google/home/gspencer/code/devtools/tool/flutter-sdk/bin/flutter pub run build_runner build --delete-conflicting-outputs
```

Reply:
Ahh, if I pass `--upgrade` then it forces it to do the pub get.